sql-join/what-and-when.sql

```diff
@@ -0,0 +1,12 @@
+SELECT "f"."title",
+  "f"."releaseYear",
+  "g"."genre"
+FROM "films" AS "f"
+JOIN (SELECT "genreId",
```

Comment:
This one and the next could also be simplified, but you know what you are doing so I won't belabor the point.